routing/car_router.cpp

```diff
@@ -281,24 +281,28 @@ bool CarRouter::FindRouteMSMT(TFeatureGraphNodeVec const & sources,
           FindSingleRouteDispatcher(sourceEdge, targetEdge, delegate, mapping, route);
       switch (code)
       {
-      case NoError: return true;
-      case Cancelled: return false;
+      case StartPointNotFound:
+      case EndPointNotFound:
+      case RouteNotFound:
+        continue;
+
+      case NoError:
+      case Cancelled:
+      case InconsistentMWMandRoute:
+      case RouteFileNotExist:
+      case PointsInDifferentMWM:
```

Comment:
If points are in different mwm, we definitely want to search cross-mwm path, isn't it?